src/backend/distributed/commands/truncate.c

```diff
@@ -267,14 +267,14 @@ ErrorIfUnsupportedTruncateStmt(TruncateStmt *truncateStatement)
 
 		ErrorIfIllegallyChangingKnownShard(relationId);
 
+#if PG_VERSION_NUM < PG_VERSION_14
```

Comment:
Marco suggested that we let this go through and let PG give the proper error. I think our message might be nicer, but for users it is much easier to find online the other message:
```SQL
truncate foreign_table;
ERROR:  "foreign_table" is not a table
```

Or, in pg 14, if you try to truncate an fdw that does not support TRUNCATE such as file_fdw, you get
```SQL
ERROR:  cannot truncate foreign table "file_test"
```